code/modules/clothing/clothing.dm

```diff
@@ -56,6 +56,25 @@
 		if(M.putItemFromInventoryInHandIfPossible(src, H.held_index))
 			add_fingerprint(usr)
 
+/obj/item/reagent_containers/food/snacks/clothing
+	name = "oops"
+	desc = "If you're reading this it means I messed up. This is related to moths eating clothes and I didn't know a better way to do it than making a new food object."
+	list_reagents = list("nutriment" = 1)
+	tastes = list("dust" = 1, "lint" = 1)
+
+/obj/item/clothing/attack(mob/M, mob/user, def_zone)
+	if(user.a_intent != INTENT_HARM && ismoth(M))
+		var/obj/item/reagent_containers/food/snacks/clothing/clothing_as_food = new
+		clothing_as_food.name = name
+		//clothing_as_food.attack(M, user, def_zone) <--- this errored and said some shit about types

```

Comment:
Maybe it needs to be `M.attack(clothing_as_food, etc)` ?

Reply:
attack is a member of /obj/item though which is the weird part?

Reply:
Actually wait I think this was fixed when I actually defined /food/snacks/clothing, recompiling to see if it fixed itself.

Reply:
Yeah it was, I'll test then push.